src/main.zig

```diff
@@ -1048,7 +1048,7 @@ fn buildOutputType(
                         try cssan.addIncludePath(.iframework, arg, args_iter.nextOrFatal(), false);
                     } else if (mem.eql(u8, arg, "--version")) {
```

Comment:
The docs for this is "Dynamic library **semver**" which wasn't actually true. Now it is because it no longer accepts an omitted patch component (according to the spec "1.0" is invalid). This might cause breakage.